members/C000984.yaml

```diff
@@ -69,46 +71,14 @@ contact_form:
             Casework Concerns: Casework
             Scheduling Requests: Scheduling_Requests
             Tour Requests: Tour_Requests
-        - name: legislation_concerns
-          selector: "#legislation_concerns"
-          value: $TOPIC
-          required: true
-          options:
-            Please select One: " "
-            Agriculture: AGR
-            Animals: ANI
-            "Arts/Entertainment/Humanities ": ART
-            "Banking\\Economy": BAN
-            "Budget/Appropriations\\Economy": BUD
-            Campaign Finance Reform: CAMFIN
-            Congress and Elections: CONG
-            "Defense\\Military": DEF
-            Education: EDU
-            Energy/Nuclear: ENG
-            Environment: ENV
-            Federal Employees: FED
-            Foreign Relations: FOR
-            Guns: GUNS
-            Health Issues: HEALTH
-            Immigration: IMM
-            "Judiciary, court issues": JUD
-            Labor Issues: LBR
-            LGBT: LGBT
-            Postal: POS
-            Telecommunications/Broadcasting/TV: TEC
-            Social Services: SOCISERV
-            Tax Issues: TAX
-            Transportation: TRANS
-            Trade Issues: TRADE
-            Veterans: VET
     - click_on:
         - value: Send Email
           selector: "#submit"
     - find:
-        - value: The following information has been submitted
+        - value: "your message has been sent"
```

Comment:
That's the way it's suppose to look, but it will be pretty easy to find a paragraph element. Is find really needed? If so, it should find something unique to the success page in this case.